### DISCUSSION TITLE
Worked Amazing but...

### DISCUSSION THREAD

**Discussion by incompletebiped** [ITDev93/SHM-MOD#2] [General]
I am stuck on the ECG Tile Enabler & Linker. I have a Galaxy Watch 6 so it is WearOS but it says Cannont install ECG Tile Link because one of the two is installed (Tizen SHM MOD or Stock SHM). I think I installed them on my phone awhile back from a previous watch. Is there anything I can do to remove them? I uninstalled them from the phone app list and reset the watch but that didn't work.

**Comment by ITDev93** [comment]
> I am stuck on the ECG Tile Enabler & Linker. I have a Galaxy Watch 6 so it is WearOS but it says Cannont install ECG Tile Link because one of the two is installed (Tizen SHM MOD or Stock SHM). I think I installed them on my phone awhile back from a previous watch. Is there anything I can do to remove them? I uninstalled them from the phone app list and reset the watch but that didn't work.

If it says it can't, then your phone still has an app with package name (com.samsung.android.shealthmonitor), which is either Tizen SHM or Stock SHM... the watch has nothing to do with that... find and uninstall the app causing this conflict and re-open the shm mod companion and you should see install button for ECG Tile Enabler

**Reply by incompletebiped** [reply]
Thank you!
